### PR TITLE
add sdkman for java8 support

### PR DESCRIPTION
**Pull request by staskolauren**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     uses: Dwolla/jenkins-agents-workflow/.github/workflows/build-docker-image.yml@main
     with:
       IMAGE_NAME: dwolla/jenkins-agent-sbt
-      TAG_NAME: CORE_TAG_JDK8_JDK11
```
Comment (staskolauren) [inline .github/workflows/ci.yml:13]: Removing `CORE_TAG_JDK8_JDK11`, in order to just build for jdk11. [Jenkins no longer supports jdk8](https://www.jenkins.io/blog/2022/06/28/require-java-11/), so there is no need to build this agent for both jdk8 and jdk11 anymore.

File: Dockerfile [hunk 2]
```diff
@@ -21,4 +21,8 @@ COPY --from=sbt-plugins-cache /root/.sbt ${JENKINS_HOME}/.sbt
 USER root
 RUN chown -R jenkins ${JENKINS_HOME}
 
+RUN export SDKMAN_DIR="${JENKINS_HOME}/.sdkman" && curl -s "https://get.sdkman.io" | bash
```
Comment (bpholt) [inline Dockerfile:24]: This is probably fine for now, but pulling sdkman in this way reduces the repeatability of this build. Is there a way to grab a specific version?

**Comment by bpholt** [comment]
Do we need to update any downstream builds before merging this, since we will now only be publishing one tag?

**Comment by staskolauren** [comment]
> Do we need to update any downstream builds before merging this, since we will now only be publishing one tag?

Actually I think I just realized this image isn't used as the base image for any downstream images, so hopefully that means we have one less step to do! We use [jenkins-agent-nvm as the base image](https://github.com/Dwolla/jenkins-agent-docker-nvm-sbt/blob/main/Dockerfile#L4) for `jenkins-agent-nvm-sbt`, which I think is where I was getting confused.

**Comment by calebwsutton** [comment]
> > Do we need to update any downstream builds before merging this, since we will now only be publishing one tag?
> 
> Actually I think I just realized this image isn't used as the base image for any downstream images, so hopefully that means we have one less step to do! We use [jenkins-agent-nvm as the base image](https://github.com/Dwolla/jenkins-agent-docker-nvm-sbt/blob/main/Dockerfile#L4) for `jenkins-agent-nvm-sbt`, which I think is where I was getting confused.

Does this mean we need to make a similar change to the [Dockerfile](https://github.com/Dwolla/jenkins-agent-docker-nvm-sbt/blob/main/Dockerfile#L17) for `jenkins-agent-nvm-sbt`?

True! If we want to support java8 on nvm-sbt that is. I think we probably do?

File: Dockerfile (hunk 2 above)
Comment (staskolauren) [reply Dockerfile:24]: Cross posting from @aquilesg and I's thread [here](https://github.com/Dwolla/jenkins-agent-docker-nvm-sbt/pull/21#discussion_r1149458247)!

**Review by aquilesg — APPROVED** [review]

**Pull request merged**